lib/class-sp-wp-search.php

```diff
@@ -305,6 +305,17 @@ public static function wp_to_es_args( $args ) {
 						);
 
 						break;
+
+					case 'author':
+						$es_query_args['aggregations'][ $label ] = array(
+							'terms' => array(
+								'field' => $facet['field'],
```

Comment:
nah, CAP and bylines are both tax terms, so there's already a built-in facet for those